src/locale/locales/pt-BR/messages.po

```diff
@@ -5599,7 +5599,7 @@ msgstr "Resposta Oculta pelo Autor da Thread"
 #: src/components/moderation/ModerationDetailsDialog.tsx:114
 #: src/lib/moderation/useModerationCauseDescription.ts:122
 msgid "Reply Hidden by You"
-msgstr "Responder Oculto por Você"
+msgstr "Resposta oculta por você"
```

Comment:
```suggestion
msgstr "Resposta ocultada por você"
```

This phrase is the better because it uses the past participle "ocultada," which suggests that the action of hiding has already been performed and completed